main.py

```diff
@@ -15,12 +15,25 @@
 def get_data(metrics: list, tickers: list):
     fetcher = data_fetcher.DataFetcher("downloaded_data")
     fetcher.init_downloads_directory()
```

Comment:
The format which the pd.to_csv() function saves date is not compatible with numpy. Therefore it is best to be saved as %Y-%m-%d.

Reply:
Done by me :)